tokens.xml

```diff
@@ -8114,6 +8222,20 @@ Equip {0}</text>
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Storm Crow Token</name>
```

Comment:
```suggestion
            <name>Storm Crow </name>
```
the card specifies that the card is named "Storm Crow" this means it does not get Token added to the name officially, yet in order to distinguish it from Storm Crow the card we need to add a space